### DISCUSSION TITLE
Chess Game

### DISCUSSION THREAD

**Discussion by DavidDHBW** [DavidDHBW/codespaces-jupyter#9] [General]
ich habe diese Woche am Schach Spiel weiter gecoded. 
Hinzu gekommen ist zum einen die DrawBoard und DrawPieces function. Auch habe ich die Hauptschleife für main gecoded.
[code](https://github.com/DavidDHBW/codespaces-jupyter/blob/main/Python/Chess/chessMain.py)

außerdem habe ich mit ein bisschen mit Rekursion beschäftigt und eine Funktion für Fakultät geschrieben

**Comment by Domidhbw** [comment]
Hey des klingt richtig cool, ich fänds stark würdest du noch ein bild davon zeigen wie es bisher aussieht
